articles.tex

```diff
@@ -67,7 +67,7 @@
 The name of this Special Interest House shall be Computer Science House.
 
 \asection{Derivation of Authority}
-The Computer Science House shall recognize that it receives its right to function as a Special Interest House from the Department of Residence Life in conjunction with the Residence Halls Association.
+The Computer Science House shall recognize that it receives its right to function as a Special Interest House from the Rochester Institute of Technology Center for Residence Life.
```

Comment:
I would remove the "Rochester Institute of Technology" bit. It reads weird to see "the Rochester Institute of Technology Center for Residence Life." I think that within the constraints of this document "... from the Center for Residence Life." should suffice.

Nit: The Center for Residence Life's website has inconsistent capitalization of "for" which slightly bothers me.

Reply:
Agreed on removing the "Rochester Institute of Technology".